src/vs/editor/contrib/codeAction/common/types.ts

```diff
@@ -24,6 +24,7 @@ export class CodeActionKind {
 	public static readonly SourceOrganizeImports = CodeActionKind.Source.append('organizeImports');
 	public static readonly SourceFixAll = CodeActionKind.Source.append('fixAll');
 	public static readonly SurroundWith = CodeActionKind.Refactor.append('surround');
+	public static readonly Notebook = new CodeActionKind('notebook');
```

Comment:
This kind is only used in notebook, should we define this in our notebook component? It feels wrong that monaco needs to know about notebook.